docs/create_user_nailgun.py

```diff
@@ -2,10 +2,13 @@
 # -*- coding: utf-8 -*-
 """Create an identical user account on a pair of satellites."""
 from __future__ import print_function
-from nailgun.config import ServerConfig
-from nailgun.entities import Organization, User
+
 from pprint import pprint
 
+from nailgun.config import ServerConfig
+from nailgun.entities import Organization
+from nailgun.entities import User
```

Comment:
Two different classes imports from same module? Can we instruct pre-commit to make this one or is it intentional?

Reply:
This is the ordering done by reorder-python-imports, not by pre-commit itself.

https://github.com/asottile/reorder_python_imports#what-does-it-do

Reply:
Yes, Coz I don't see any advantage other than a bunch of import lines added. These days our IDE itself shows if there are any import conflicts.

But that's just my opinion and probably you would have already discussed with @JacobCallahan and @rochacbruno .

Reply:
Personally, I'm not a particular fan of splitting the imports like this, but I've learned to enjoy having the tool manage them for me, and know that they're always sorted and its always easy to locate an import when visually parsing them.

Reply:
@jyejare Rationale is given a little further down on page linked by Mike, under [Why this style](https://github.com/asottile/reorder_python_imports#why-this-style) heading. 

We don't really have that kind of problem in our repos, so it's more like a style choice. Some people will like it, some people won't. I didn't like it at first, but we all can agree that our time is better spent elsewhere. Having tool managing it for us means we don't have to waste a single second thinking about this.